tests/utils/structure_tester.py

```diff
@@ -79,20 +77,23 @@ class StructureTester:
         "BEDROCK_J2": AmazonBedrockPromptDriver(
             model="ai21.j2-ultra", prompt_model_driver=BedrockJurassicPromptModelDriver()
         ),
-        "SAGEMAKER_LLAMA_7B": AmazonSageMakerPromptDriver(
-            model=os.environ["SAGEMAKER_LLAMA_ENDPOINT_NAME"],
-            max_attempts=1,
-            prompt_model_driver=SageMakerLlamaPromptModelDriver(max_tokens=4096),
+        "BEDROCK_LLAMA2_13B": AmazonBedrockPromptDriver(
+            model="meta.llama2-13b-chat-v1", prompt_model_driver=BedrockLlamaPromptModelDriver(), max_attempts=1
         ),
-        "SAGEMAKER_FALCON_7b": AmazonSageMakerPromptDriver(
-            model=os.environ["SAGEMAKER_FALCON_ENDPOINT_NAME"],
-            max_attempts=1,
-            prompt_model_driver=SageMakerFalconPromptModelDriver(),
+        "BEDROCK_LLAMA2_70B": AmazonBedrockPromptDriver(
+            model="meta.llama2-70b-chat-v1", prompt_model_driver=BedrockLlamaPromptModelDriver(), max_attempts=1
         ),
+        # "SAGEMAKER_LLAMA_7B": AmazonSageMakerPromptDriver(
+        #     model=os.environ["SAGEMAKER_LLAMA_ENDPOINT_NAME"],
+        #     prompt_model_driver=SageMakerLlamaPromptModelDriver(max_tokens=4096),
+        # ),
+        # "SAGEMAKER_FALCON_7b": AmazonSageMakerPromptDriver(
+        #     model=os.environ["SAGEMAKER_FALCON_ENDPOINT_NAME"],
+        #     prompt_model_driver=SageMakerFalconPromptModelDriver(),
+        # ),
```

Comment:
Is there a better way to keep these around? Don't actively keep the infrastructure running.

Reply:
We could add an object in the form of:

```python
class TesterPromptDriverOption:
    prompt_driver: BasePromptDriver
    enabled: bool
```

and then set `enabled` to `False` for cases that we don't want to be testing all the time. This will also allow us to run it "with defaults" and "force run all."